dialog/android/src/main/java/com/capacitorjs/plugins/dialog/DialogPlugin.java

```diff
@@ -0,0 +1,101 @@
+package com.capacitorjs.plugins.dialog;
+
+import androidx.appcompat.app.AppCompatActivity;
+import com.getcapacitor.JSObject;
+import com.getcapacitor.NativePlugin;
+import com.getcapacitor.Plugin;
+import com.getcapacitor.PluginCall;
+import com.getcapacitor.PluginMethod;
+
+@NativePlugin(name = "Dialog")
+public class DialogPlugin extends Plugin {
+
+    @PluginMethod
+    public void alert(final PluginCall call) {
+        final AppCompatActivity c = this.getActivity();
+        final String title = call.getString("title");
+        final String message = call.getString("message");
+        final String buttonTitle = call.getString("buttonTitle", "OK");
+
+        if (title == null || message == null) {
+            call.reject("Please provide a title or message for the alert");
+            return;
+        }
+
+        if (c.isFinishing()) {
+            call.reject("App is finishing");
+            return;
+        }
+
+        Dialog.alert(c, message, title, buttonTitle, (value, didCancel, inputValue) -> call.resolve());
```

Comment:
rename `c` to `activity`, more descriptive

```suggestion
        final AppCompatActivity activity = this.getActivity();
        final String title = call.getString("title");
        final String message = call.getString("message");
        final String buttonTitle = call.getString("buttonTitle", "OK");

        if (title == null || message == null) {
            call.reject("Please provide a title or message for the alert");
            return;
        }

        if (activity.isFinishing()) {
            call.reject("App is finishing");
            return;
        }

        Dialog.alert(activity, message, title, buttonTitle, (value, didCancel, inputValue) -> call.resolve());
```